Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(Revision 3673)
+++ CMakeLists.txt	(Arbeitskopie)
@@ -119,7 +119,7 @@
 ################################################################################################
 # Some random user settings
 #
-ADD_DEFINITIONS(-DGLOBCONFPATH="${SYSCONF_INSTALL_DIR}/sword.conf")
+#ADD_DEFINITIONS(-DGLOBCONFPATH="${SYSCONF_INSTALL_DIR}/sword.conf")
 SET(CONFFILE "[Install]
 DataPath=${SHARE_INSTALL_PREFIX}/sword")
 FILE(WRITE "${CMAKE_CURRENT_BINARY_DIR}/sword.conf" "${CONFFILE}")
Index: include/utf8transliterator.h
===================================================================
--- include/utf8transliterator.h	(Revision 3673)
+++ include/utf8transliterator.h	(Arbeitskopie)
@@ -45,6 +45,8 @@
 #include <defs.h>
 #include <map>
 
+using icu::UnicodeString;
+
 SWORD_NAMESPACE_START
 
 class SWModule;
Index: src/modules/filters/scsuutf8.cpp
===================================================================
--- src/modules/filters/scsuutf8.cpp	(Revision 3673)
+++ src/modules/filters/scsuutf8.cpp	(Arbeitskopie)
@@ -35,8 +35,8 @@
 #include <unicode/unistr.h>
 #endif
 
+using icu::UnicodeString;
 
-
 SWORD_NAMESPACE_START
 
 
Index: src/modules/filters/utf8nfc.cpp
===================================================================
--- src/modules/filters/utf8nfc.cpp	(Revision 3673)
+++ src/modules/filters/utf8nfc.cpp	(Arbeitskopie)
@@ -30,6 +30,8 @@
 #include <utf8nfc.h>
 #include <swbuf.h>
 
+using icu::UnicodeString;
+
 SWORD_NAMESPACE_START
 
 UTF8NFC::UTF8NFC() {
Index: src/modules/filters/utf8scsu.cpp
===================================================================
--- src/modules/filters/utf8scsu.cpp	(Revision 3673)
+++ src/modules/filters/utf8scsu.cpp	(Arbeitskopie)
@@ -25,6 +25,8 @@
 #include <utf8scsu.h>
 #include <swbuf.h>
 
+using icu::UnicodeString;
+
 SWORD_NAMESPACE_START
 
 
