Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 3667)
+++ CMakeLists.txt	(working copy)
@@ -119,7 +119,7 @@
 ################################################################################################
 # Some random user settings
 #
-ADD_DEFINITIONS(-DGLOBCONFPATH="${SYSCONF_INSTALL_DIR}/sword.conf")
+#ADD_DEFINITIONS(-DGLOBCONFPATH="${SYSCONF_INSTALL_DIR}/sword.conf")
 SET(CONFFILE "[Install]
 DataPath=${SHARE_INSTALL_PREFIX}/sword")
 FILE(WRITE "${CMAKE_CURRENT_BINARY_DIR}/sword.conf" "${CONFFILE}")
